lib/supabase/in_process_payments/selectPayments.ts

```diff
@@ -18,13 +17,15 @@ export interface InProcessPaymentsQuery {
   page?: number;
   artist?: string;
   collector?: string;
+  smartAccountAddress?: string;
 }
 
 export async function selectPayments({
   limit = 20,
   page = 1,
   artist,
   collector,
+  smartAccountAddress,
```

Comment:
why is a new `smartAccountAddress` param needed?
- Why can't you use the existing `artist` and `collector` params to accomplish the same functionality?